server/src/db.js

```diff
@@ -133,3 +144,49 @@ exports.markersForArgs = function(starting, numberOfArgs) {
   }
   return result.join(", ");
 };
+
+function doNothing() {
+}
+
+function initTiming() {
+  if (!logQueryLimit) {
+    return doNothing;
+  }
+  let caller = getCallerPosition(2);
+  if (caller == "skip") {
```

Comment:
I guess I was trying to make this minimal, without adding a bunch of stuff throughout the codebase. Given that, it required some cleverness to identify queries (there might have been an option to identify queries based on the SQL, but then there's a question of how to uniquely identify SQL without putting every query into the log).

But I don't think this should be forever code... like, this code isn't Mr. Right, just Mr. Right Now.